### DISCUSSION TITLE
Chia Blockchain

### DISCUSSION THREAD

**Discussion by EvgKrsk24** [Chia-Network/chia-blockchain#8611] [Support]
In the Chia Blockchain program, I cannot get the Bind Login Link for https://www.ihpool.com.
In the pool section I add poll. You need XCH to join a pool -> on https://faucet.chia.net/ -> I enter https://chiapp.hpool.com -> address appears invalid. How to get Bind Login Link for https://www.ihpool.com?

**Comment by loppefaaret** [comment]
you would have to contact hpool about this issue

**Comment by BrandtH22** [comment]
@EvgKrsk24 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
